frontend/src/pages/analytics/retention/AnalyticsRetention.tsx

```diff
@@ -535,11 +535,11 @@ const AnalyticsRetention: React.FC<AnalyticsRetentionProps> = (
                   });
                 }}
                 changeStartEvent={(index, item) => {
-                  const eventName = item?.value;
+                  const eventName = item?.name;
                   const eventParameters = getEventParameters(eventName);
                   const parameterOption = parametersConvertToCategoryItemType(
```

Comment:
Consider set the second parameter of the `parametersConvertToCategoryItemType` with default value like `[]`, there are too many `eventParameters ?? []`.

Reply:
fixed.